### PR TITLE
Cleaner whitespace

### PR DESCRIPTION
**Pull request by walterdavis**
I read your article and tried it out. One thing that bothered me were all the empty lines in the generated service. ERB has a great feature to suppress those, so I put it to use. I also added the magic comment for modern Ruby 2 conventions, so Rubocop won't complain and add it.

### CODE REVIEW

**Comment by ivanmatas** [comment]
@walterdavis Thank you so much for you contribution! You are absolutely right about this. I will merge it, I just have some question about indentation that I will leave as a comment in the PR. 

Once again, I appreciate your great remarks :)

File: lib/matas_service_generator/service/templates/service.erb [hunk 1]
```diff
@@ -1,18 +1,18 @@
-<% if @module_name.present? %> module <%= @module_name.camelize %>  <% end %>
-class <%= class_name %>
+<%- indent = @module_name.present? ? '  ' : '' -%>
```
Comment (ivanmatas) [inline lib/matas_service_generator/service/templates/service.erb]: The changes you made look good for a service in a module with methods, but If I make a simple service, the indentation of the 'call' and 'initialize' does not seem right. What do you think of adding a second level indentation as stated down in this comment, and using that second_level_indentation everywhere except for 'class_name' and it's ending? If you have better idea, please do share. I wasn't aware of this in erb, but now I would like to make it look nice for each scenario :D 

```suggestion
<%- indent = @module_name.present? ? '  ' : '' -%>
<%- second_level_indent = indent + '  '  -%>
```

Comment (walterdavis) [reply lib/matas_service_generator/service/templates/service.erb]: Hmmm. I have tried it in all the documented combinations: with and without a module, with and without additional methods, and the only thing I see that looks wrong is the line-spacing around `private`. I need to add a newline after private to go with the one before. All the indents are two spaces, as per the Rubocop standard, and when you're inside a module, it's four spaces.

**Comment by walterdavis** [comment]
I'm going to go out on a limb here and guess that you are using tabs in your editor. That's another beef that Rubocop will have with you...

**Comment by ivanmatas** [comment]
You're right, tabs were the issue. Okay, I'm merging this, thanks a lot for your help! 👍

**Comment by walterdavis** [comment]
I just pushed a couple more changes, I started looking deeper into the code and found some more opportunities for improvement. Please let me know if you'd like a separate MR for those, as they stray from the topic of whitespace.

**Comment by ivanmatas** [comment]
It will be fine as is regarding separate PRs. Great changes, it really improved readability and cleaned up the code real nice. It will be merged in few minutes.

**Pull request merged**